array-manipulation-1/src/compact.js

```diff
@@ -0,0 +1,17 @@
+/* exported compact */
+
+const years = [2020, 2021, 2023, null, 1992, -1992];
+
+function compact(array) {
+  const result = [];
+
+  for (let i = 0; i < array.length; i++) {
+    const test = array[i];
+    if (test) {
+      result.push(test);
```

Comment:
`test` isn't a very good variable name. Try using names that are more descriptive about what value they have